iteration/iteration_test.go

```diff
@@ -458,84 +450,51 @@ func (s *TestIterationRepository) TestExistsIteration() {
 
 func (s *TestIterationRepository) TestIsActive() {
 	t := s.T()
+	newSpace := space.Space{
+		Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+		OwnerId: s.testIdentity.ID,
+	}
+	repoSpace := space.NewRepository(s.DB)
+	space, err := repoSpace.Create(context.Background(), &newSpace)
+	assert.Nil(t, err)
 	t.Run("user active is true", func(t *testing.T) {
-		// given
-		newSpace := space.Space{
-			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
-			OwnerId: s.testIdentity.ID,
-		}
-		repoSpace := space.NewRepository(s.DB)
-		space, err := repoSpace.Create(context.Background(), &newSpace)
-		assert.Nil(t, err)
-
 		level0IterationName := "Top level iteration"
-		trueUserActive := true
 		i1 := iteration.Iteration{
 			Name:       level0IterationName,
 			SpaceID:    space.ID,
-			UserActive: &trueUserActive,
+			UserActive: true,
```

Comment:
KEEP THIS LINE ;)

Reply:
this line got removed as textfixture was introduced